redisinsight/main.dev.ts

```diff
@@ -119,8 +119,20 @@ const bootstrap = async () => {
     trayInstance = tray.buildTray();
   }
 
-  if (process.env.NODE_ENV === 'production') {
-    new AppUpdater();
+  let upgradeUrl;
+  switch (process.env.NODE_ENV) {
```

Comment:
I think better to replace this `switch` to rewrite env variable in the config for circle.ci